src/bm_utils.c

```diff
@@ -108,7 +108,10 @@ static char alpha_num[65] =
 #define PROTO(s) ()
 #endif
 
-char     *getenv PROTO((const char *name));
+#ifndef STDLIB_HAS_GETENV
```

Comment:
If we're _requiring_ this to exist in the CMake phase, we don't need this check here.